packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -907,6 +907,32 @@ export class SamplerOperations {
         });
     }
 
+    public getCoFiXSellQuotes(
+        makerToken: string,
+        takerToken: string,
+        takerFillAmounts: BigNumber[],
+    ): SourceQuoteOperation {
+        return new SamplerContractOperation({
+            source: ERC20BridgeSource.CoFiX,
+            contract: this._samplerContract,
+            function: this._samplerContract.sampleSellsFromCoFiX,
+            params: [takerToken, makerToken, takerFillAmounts],
+        });
+    }
+
+    public getCoFiXBuyQuotes(
+        makerToken: string,
+        takerToken: string,
+        makerFillAmounts: BigNumber[],
+    ): SourceQuoteOperation {
+        return new SamplerContractOperation({
+            source: ERC20BridgeSource.CoFiX,
+            contract: this._samplerContract,
+            function: this._samplerContract.sampleBuysFromCoFiX,
+            params: [takerToken, makerToken, makerFillAmounts],
```

Comment:
Probably want to manually parse this (see `callback`) and extract the `pool` in the FillData.